.github/workflows/transifex_merge_translations.yml

```diff
@@ -0,0 +1,37 @@
+# Since transifex-synchronization-source is just a staging branch to control
+# updates to Transifex, we need to force merge the translations to master,
+# excluding the English source files
+name: Translations merge to master
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - transifex-synchronization-source
+
+jobs:
+  merge-translations:
+    if: github.event.pull_request.user.login == 'transifex-integration[bot]' && github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master branch
+        uses: actions/checkout@v4
+        with:
+          ref: master
+
+      - name: Setup Git User
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+      - name: Force merge translation directories (excluding English)
+        run: |
+          git fetch --unshallow origin transifex-synchronization-source
+          for dir in language/*/; do
```

Comment:
What is the expected behavior when new language was added on `transifex-synchronization-source` or removed on `transifex-synchronization-source`?

Reply:
A new language should be merged to master. Removing a translated language that has been added to the game is not an expected action we will ever take, so if it happens we would have to sort that out manually on a case-by-case basis.

Reply:
> A new language should be merged to master.

The current implementation is not doing that, it's only updating languages that already exists, because it iterates over list of languages on the master branch.

So, telling you about this now 3rd time: https://discord.com/channels/549281623154229250/549282166543089674/1287886592010223686